src/routes/OrganizationRoutes.jsx

```diff
@@ -14,14 +14,46 @@ import FundingOrgsIndex from "../Components/FundingOrgs/FundingOrgsIndex";
 import GrantCopy from "../Components/Grants/GrantCopy";
 import GrantEdit from "../Components/Grants/GrantEdit";
 import GrantShow from "../Components/Grants/GrantsShow";
+import GrantShowOverview from "../Components/Grants/GrantShowOverview";
 import GrantsIndex from "../Components/Grants/GrantsIndex";
 import GrantsNew from "../Components/Grants/GrantsNew";
 import ReportsNew from "../Components/Reports/ReportsNew";
 import ReportsShow from "../Components/Reports/ReportsShow";
 import RedirectToDashboard from "../Components/Helpers/RedirectToDashboard";
 import UserIndexPage from "../pages/UserIndex/UserIndexPage";
+import {
+  DndContext,
+  PointerSensor,
+  useSensor,
+  useSensors,
+  closestCenter,
+} from "@dnd-kit/core";
+import { arrayMove } from "@dnd-kit/sortable";
+import {
+  restrictToFirstScrollableAncestor,
+  restrictToVerticalAxis,
+} from "@dnd-kit/modifiers";
 
 export default function OrganizationRoutes() {
+  const sensors = useSensors(useSensor(PointerSensor));
+  const [sortableSections, setSortableSections] = useState([]);
+  const [activeId, setActiveId] = useState(null);
+
+  function handleDragStart(event) {
+    const { active } = event;
+    setActiveId(active.id);
+  }
+
+  function handleDragEnd({ active, over }) {
+    if (active.id !== over.id) {
+      setSortableSections((items) => {
+        const oldIndex = items.findIndex((item) => item.id === active.id);
+        const newIndex = items.findIndex((item) => item.id === over.id);
+        return arrayMove(items, oldIndex, newIndex);
+      });
+    }
+  }
+
```

Comment:
I'm wondering if we could find a way to move the dnd specific stuff outside of the OrganizationRoutes component to keep it contained specific to the grant show overview page. Is it possible we can just move this into that component or create an extra wrapper component around it so that the organization routes file just has routing things?

Reply:
We decided to leave this refactor for the end of the feature too - at minimum, we should be able to move the handlers into a separate file.